src/test/java/ludogorie_soft/reservations_platform_api/mapper/UserMapperTest.java

```diff
@@ -0,0 +1,76 @@
+package ludogorie_soft.reservations_platform_api.mapper;
+
+import ludogorie_soft.reservations_platform_api.dto.RegisterDto;
+import ludogorie_soft.reservations_platform_api.entity.User;
+import ludogorie_soft.reservations_platform_api.helper.UserTestHelper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.modelmapper.ModelMapper;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Setup IntelliJ to avoid wildcard imports https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Reply:
Done!